### PR TITLE
[Doc Improvement][Usage of F5 key in Teams Toolkit][2307590]

### PR DESCRIPTION
**Pull request by v-bdeepa**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascriptF5.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/05616e501eaac0e298e08a5e7de55b685e7a2f0f/msteams-platform/sbs-gs-javascriptF5.yml) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascriptF5.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b308267aba62847ce0ce2643d36847fcc627394a/msteams-platform/sbs-gs-javascriptF5.yml) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascriptF5.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/8930466c83206c2cf7e116be8bea4dbd2c51692f/msteams-platform/sbs-gs-javascriptF5.yml) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascriptF5.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/3ae5fa7555f348db13135dd77e5df984fbd22d43/msteams-platform/sbs-gs-javascriptF5.yml) | 91 | 99 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0c3bf117-98f1-42ea-80a8-e49241fe948e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascriptF5.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b7e9cdc3ead265c5faae9c93ad67533ae9bd955d/msteams-platform/sbs-gs-javascriptF5.yml) | 91 | 99 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/37b44337-5dde-43ec-b088-867fe1d028bb) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascriptF5.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/2add1fb14f3129db0724fb0dbef9eba99990c677/msteams-platform/sbs-gs-javascriptF5.yml) | 91 | 99 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/55b5b2a1-0838-4c28-8531-e452eb634c1f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascriptF5.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/1887b8cfdb6490029f5232ef929a9fcc336d96e8/msteams-platform/sbs-gs-javascriptF5.yml) | 91 | 99 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4c3657a9-11d6-4ceb-9b9a-b26763932946) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascriptF5.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/e7789eaf1f32dd671c65345f5b3cdbcecbde7ce5/msteams-platform/sbs-gs-javascriptF5.yml) | 91 | 99 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cd680502-d269-416e-bb15-551d5c554a9d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascriptF5.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/262f25084c17640167f64b59089a7aa0c25ff9fb/msteams-platform/sbs-gs-javascriptF5.yml) | 92 | 92 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4d5b3917-9507-40cf-b893-346406d7f6b2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascriptF5.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/04fcb7416434e7edc44d9ee67f73b6e57bd5a6c2/msteams-platform/sbs-gs-javascriptF5.yml) | 92 | 92 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6ce7513a-17f1-4236-b50e-29c6e3ae69c7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascriptF5.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/47f6663fd6d4bfe99e53202986c9b76517eceb18/msteams-platform/sbs-gs-javascriptF5.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a9628610-0983-4404-ae52-d419609c3585) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascriptF5.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/4f741cae1b6a52b3fdaf0e8e489644a55476820a/msteams-platform/sbs-gs-javascriptF5.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a7c51566-5d59-4953-9783-22147b53152b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascriptF5.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/bdb4c7d0194812dd0765f18abe1e224d074965fc/msteams-platform/sbs-gs-javascriptF5.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5fdb3b5a-aae1-4021-9002-db4e5ede00fa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/00419e8690d1ad142ffc727a8b9f47b05ef804e5/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a8df92f7-3c06-404e-8ad1-898c8ac1bb83) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/a25527e9422685565da3227ffbd74742461870ff/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6d34b859-6635-4bea-8076-a6eac36c0754) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a25527e9422685565da3227ffbd74742461870ff/msteams-platform/whats-new.md) | 92 | 36 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6d0f476c-6395-49d1-8c13-d06d5d27b843) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/61fcd843a5105a443ce07e63eb7290863d748585/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ccaa99fc-cc45-4b06-936b-8b3854f082ca) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/04ddc0d5038f4a4ffee05d32260820472776169e/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 89 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/799ae551-73a6-4345-a8d1-4067e0210551) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/c7358e876821655377ff055f07076073779a271f/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 89 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/37de58e5-2ee7-43bb-9881-e07e398fc95f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/dbbd9413e89e90ddf86239a96fa658c9d34b6e22/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c1824397-a6a1-489d-9d6f-31820597fda8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-gs-javascript-new-debug.yml [hunk 1]
```diff
@@ -0,0 +1,1114 @@
...
+       4. Registers a bot in Bot Framework and configures the bot.
+       5. Registers a Teams app in Teams Developer Portal and configures the app.
+       6. Starts the bot app.
+       7. Starts Teams in a web browser with a command to instruct Teams to sideload the Teams app.
+
+        </details>
+
+        :::image type="content" source="../msteams-platform/assets/images/teams-toolkit-v2/first-bot/f5-build-and-run.png" alt-text="Screenshot showing when F5 key is pressed for debugging.":::
+
+        > When you run the app for the first time, all dependencies are downloaded, and the app is built. A browser window automatically opens when the build is complete. This process can take 3-5 minutes to complete.
```
Comment (kuojianlu) [inline msteams-platform/sbs-gs-javascript-new-debug.yml:630]: Miss `> [!NOTE]`.

File: msteams-platform/sbs-gs-javascript-new-debug.yml [hunk 2]
```diff
@@ -0,0 +1,1114 @@
...
+        <details>
+        <summary>Learn what happens when you run your app locally in the debugger.</summary>
+
+        When you select **F5**, Teams Toolkit:
+
+        1. Checks all the following prerequisites:
+          * Supported Node.js version is installed.
+          * Microsoft 365 account is logged in and sideloading enabled. 
+          * Ngrok is installed.           
+          * NPM packages for bot app are installed.
```
Comment (kuojianlu) [inline msteams-platform/sbs-gs-javascript-new-debug.yml:866]:
```suggestion
          * NPM packages for messaging extension app are installed.
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/4845ea9846d824fca70f5663e71c52636ab7aba3/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d08b78fa-6f7f-4f40-a7a2-8521e022c696) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/44527b419de96b86340c97fb07139a25541fadab/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3880276a-690f-4419-9cff-e0ae4e7722e2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/c9c27e3f397fb18565e3142e730fbe1d53b54a29/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9b6aed43-2efa-4144-9ad9-0b0e4ab865c9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/7ecb45dfa625be71edf5a22c87b6488259287daa/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/509d609e-c9ab-43a9-ba0b-eba936de1879) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/98ba509e79884afa42dc6e665a6396cf4e085ff6/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dbe822c5-dd59-4006-916c-646c78e02aa2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/2e0c0f5dd6ea4c5e23f5d8c4d01ff32a8e2fbfab/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/658d4ed2-35d2-447e-866d-460b01470d69) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/e90aca40206b7969a518610ecd17d5fc831a0247/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/76d12d0f-d376-4e5a-9c5a-6083ee43f7d0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/188365a40f10581ead58621ed749177a22641e27/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dfaa0a18-9c6f-475b-bda5-4ae7f77496d9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/51095161950b1f616ea92e709a093610856a598b/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d2403a75-7f04-415c-b174-3fd4d721cef6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript-new-debug.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/8b8feccc44950359b29e900842cd0acd479c6320/msteams-platform/sbs-gs-javascript-new-debug.yml) | 92 | 88 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b26662ba-ef74-47f3-98f0-353298430a66) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approved

**Review by kuojianlu — APPROVED** [review]

**Pull request merged**